t/19_console_redirection.t

```diff
@@ -50,16 +50,17 @@ subtest '[connect_target_to_serial]' => sub {
     $redirect->redefine(script_output => sub { return 'castleinthesky'; });
     set_var('QEMUPORT', '1988');
 
-    connect_target_to_serial(destination_ip => '192.168.1.1', ssh_user => 'Totoro');
```

Comment:
my 2 cents, leave `subtest '[connect_target_to_serial]'` as it is and create a new one named `subtest '[connect_target_to_serial] ssh_user'` that only has one check `ok(grep(/sudo su \-/, @ssh_cmd), 'Check root switching command');`

Reply:
Yes, good idea. Just pushed new changes.